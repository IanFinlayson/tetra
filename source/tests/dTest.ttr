def main():
  m = {"frank":6}
  print("set m = {'frank':6} and m = ")
  print(m)
  print("\n")
  n = {"harry":7}
  print("set n = {'harry':7} and n = ")
  print(n)
  print("\n")
  o = {m:n}
  print("set o = {m:n} and o = ")
  print(o)
  print("\n")
  m["frank"] = 2
  print("set m['frank'] = 2 and m = ")
  print(m)
  print("\n")
  print("now o = ")
  print(o)
  print("\n")
