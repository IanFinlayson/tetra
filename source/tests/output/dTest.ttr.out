set m = {'frank':6} and m = {'frank'}
set n = {'harry':7} and n = {'harry'}
set o = {m:n} and o = {{'frank'}}
set m['frank'] = 2 and m = {'frank'}
now o = {{'frank'}}

